service/history/queueProcessor.go

```diff
@@ -330,7 +330,8 @@ ProcessRetryLoop:
 			return
 		}
 
-		logging.LogOperationPanicEvent(logger, "Retry count exceeded for transfer task", err)
+		p.metricsClient.IncCounter(p.options.MetricScope, metrics.TaskStuck)
+		logging.LogOperationStuckEvent(logger, "Retry count exceeded for transfer task", err)
```

Comment:
Can we loose task by converting fatal into error?

Reply:
what do you mean by lose?